src/EnergyPlus/DataRuntimeLanguage.hh

```diff
@@ -389,7 +390,8 @@ namespace DataRuntimeLanguage {
 			Variable( 0 ),
 			Expression( 0 ),
 			TrendVariable( false ),
-			TrendVarPointer( 0 )
+			TrendVarPointer( 0 ),
+			initialized( false )
 		{}
```

Comment:
I think what you are looking in your other [comment](https://github.com/NREL/EnergyPlus/pull/5660#discussion-diff-65465781) is a call to the member constructor, where values are passed in as actual arguments, which is defined [here](https://github.com/NREL/EnergyPlus/blob/e65af6c503f9577d6fa0dc3af433064bca2d6975/src/EnergyPlus/DataRuntimeLanguage.hh#L396).  

The line you are commenting [here](https://github.com/NREL/EnergyPlus/blob/e65af6c503f9577d6fa0dc3af433064bca2d6975/src/EnergyPlus/DataRuntimeLanguage.hh#L386) is a default constructor that doesn't actually take any arguments, but instead, the class member variables are given default values in the constructor's _initializer list_.

A subtle difference.  Let me know if it's still funny.